dgf/templates/dgf/friends_header.html

```diff
@@ -1,9 +1,5 @@
 {% load sekizai_tags static %}
 
-{% addtoblock "css" %}
-    <link rel="stylesheet" type="text/css" href="{% static 'friends_header.css' %}">
```

Comment:
Esto lo he movido al base para que esté to juntito. El header se muestra siempre